pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -0,0 +1,63 @@
+//go:build linux && rego
+// +build linux,rego
+
+package securitypolicy
+
+import (
+	"context"
+	"errors"
+
+	"github.com/Microsoft/hcsshim/internal/log"
+	oci "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const regoEnforcer = "rego"
+
+func init() {
+	registeredEnforcers[regoEnforcer] = CreateRegoEnforcer
+	defaultEnforcer = regoEnforcer
```

Comment:
We are defining this here to guarantee it runs after all the `var`s are defined, so theres no race condition for what value ends up in `defaultEnforcer`, right?
Can we have that in a comment, just so it isn't moved out?